pkgs/development/libraries/libgdata/default.nix

```diff
@@ -10,8 +10,6 @@ stdenv.mkDerivation rec {
     sha256 = "0fj54yqxdapdppisqm1xcyrpgcichdmipq0a0spzz6009ikzgi45";
   };
 
-  NIX_CFLAGS_COMPILE = "-I${gnome3.libsoup.dev}/include/libsoup-gnome-2.4/ -I${gcr}/include/gcr-3 -I${gcr}/include/gck-1";
```

Comment:
How so that this is no longer needed?

Reply:
The build warned a bunch about `${gcr}/include` not existing (it in dev now). So seeing that the line was pretty old and didn't work as intended I tried removing it to see if the build worked without, so I assume it has been fixed upstream. Should've said so in the commit.